runtime/weak.c

```diff
@@ -30,7 +30,7 @@
 #include "caml/weak.h"
 
 value caml_dummy[] =
-  {(value)Make_header(0,Abstract_tag, NOT_MARKABLE),
+  {(value)Make_header(0, Abstract_tag, NOT_MARKABLE),
```

Comment:
```suggestion
  {(value)Make_header(0,Abstract_tag, NOT_MARKABLE),
```
Please avoid touching files (and ideally, lines) that don't *need* to be changed.